central/declarativeconfig/manager_impl_test.go

```diff
@@ -377,3 +362,23 @@ permissionSet: permission-set
 `),
 	})
 }
+
+func TestVerifyUpdaters(t *testing.T) {
+	m := newTestManager(t)
+	m.updaters = map[reflect.Type]updater.ResourceUpdater{
+		types.PermissionSetType: nil,
```

Comment:
I didn't realize that `nil` would be treated as a valid `ResourceUpdater` - perhaps we should add extra check to `verifyUpdaters`?